frontend/src/components/school/SchoolList.vue

```diff
@@ -162,7 +162,9 @@
             class="elevation-1 rounded"
             hide-default-header
             mobile-breakpoint="0"
+            hea
```

Comment:
Just curious what `hea` is?

Reply:
I am guessing accidental typing! I'll clean it up